src/utils/Util.js

```diff
@@ -8,12 +8,23 @@ const remote = electron.remote;
 const dialog = remote.dialog;
 const app = remote.app;
 
+// Docker Machine exits with 3 to differentiate pre-create check failures (e.g.
+// virtualization isn't enabled) from normal errors during create (exit code
+// 1).
+const precreateCheckExitCode = 3;
+
 module.exports = {
+  errPrecreateCheckFailure:  'Pre-create check failed: ',
   execFile: function (args, options) {
+    var that = this;
     return new Promise((resolve, reject) => {
       child_process.execFile(args[0], args.slice(1), options, (error, stdout) => {
         if (error) {
-          reject(new Error('Encountered an error: ' + error));
+          if (error.code === precreateCheckExitCode) {
+            reject(new Error(that.errPrecreateCheckFailure + error));
+          } else {
+            reject(new Error('Encountered an error: ' + error));
```

Comment:
Simple return `error` here via `reject(error)`. Then check `if (error.code && error.code === 3)` in the `SetupUtil` code.